src/data/list/basic.lean

```diff
@@ -1327,6 +1327,17 @@ lemma last_eq_nth_le : ∀ (l : list α) (h : l ≠ []),
 | (a :: b :: l) h := by { rw [last_cons, last_eq_nth_le (b :: l)],
                           refl, exact cons_ne_nil b l }
 
+lemma nth_le_length_sub_one {l : list α} (h₁ h₂) :
+  l.nth_le (l.length - 1) h₂ = l.last h₁ :=
```

Comment:
Can you prove `h₁` in terms of `h₂`? Note how `last_eq_nth_le` does the reverse.